styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -0,0 +1,38 @@
+{% block skipNav %}
+  {% include "@atoms/skip-nav.twig" %}
+{% endblock %}
+
+<div class="container">
```

Comment:
Are you planning on having a `div.container` around every layout? If so, this should be documented somewhere (`.md`?)

Reply:
All our templates should have a max-width which what is in SG1. Should this be in the md or implementation docs? I don't know.